docs/src/components/CodeBlock.tsx

```diff
@@ -19,11 +19,13 @@ import * as Icons from '@marigold/icons';
 import { CopyButton } from './CopyButton';
 import { ShowHideButton } from './ShowHideButton';
 import { useThemeSwitch } from './ThemeSwitch';
+import { ComponentPreview } from './ComponentPreview';
 
 enum ActionType {
   Preview = 'preview',
   Live = 'live',
   Code = 'code',
+  Demo = 'demo',
```

Comment:
to be honest I don't understand anymore when to use which ActionType.. can you add a one-line comment to each of these?

Reply:
Do you have any ideas for better naming?

Reply:
No because I don't know the differences :see_no_evil:

Reply:
what is the difference between `preview` and `live`?

Reply:
Ok bad description. Live ist the live coding example where you can type in the code box.

Reply:
I just don't see why we need these 4 different kinds, because it's not bad to always edit the code, or to always be able to copy the code :)

Reply:
Ok yes, we can also always edit or copy the code

Reply:
I will make some changes and push something new stuff :)